src/components/Observations/ObsCardStats.js

```diff
@@ -1,74 +1,72 @@
 // @flow
 
+import classnames from "classnames";
 import checkCamelAndSnakeCase from "components/ObsDetails/helpers/checkCamelAndSnakeCase";
-import { ActivityCount, QualityGradeStatus } from "components/SharedComponents";
-import { View } from "components/styledComponents";
+import { ActivityCount, Body3, QualityGradeStatus } from "components/SharedComponents";
+import { Image, View } from "components/styledComponents";
 import type { Node } from "react";
 import React from "react";
 import { useTranslation } from "react-i18next";
 import { useTheme } from "react-native-paper";
 
 type Props = {
   item: Object,
-  type?: string,
   layout?: string
 }
-
-const ObsCardStats = ( { item, type, layout }: Props ): Node => {
+/* eslint-disable react-native/no-inline-styles */
+const ObsCardStats = ( { item, layout }: Props ): Node => {
   const { t } = useTranslation( );
   const theme = useTheme( );
   const qualityGrade = checkCamelAndSnakeCase( item, "qualityGrade" );
 
-  const setIconColor = ( ) => {
+  const getIconColor = ( ) => {
+    if ( layout === "grid" ) {
+      return theme.colors.onSecondary;
+    }
     if ( item.viewed === false ) {
       return theme.colors.error;
-    } if ( layout === "grid" ) {
-      return theme.colors.onPrimary;
     }
     return theme.colors.primary;
   };
 
-  const renderIdRow = ( ) => (
-    <ActivityCount
-      count={item.identifications?.length}
-      color={setIconColor( )}
-      accessibilityLabel={t( "Number-of-identifications" )}
-      testID="ActivityCount.identificationCount"
-    />
-  );
-
-  const renderCommentRow = ( ) => (
-    <ActivityCount
-      count={item.comments?.length}
-      color={setIconColor( )}
-      accessibilityLabel={t( "Number-of-comments" )}
-      testID="ActivityCount.commentCount"
-    />
-  );
-
-  const renderQualityGrade = ( ) => (
-    <QualityGradeStatus qualityGrade={qualityGrade} color={setIconColor( )} />
-  );
-
-  const renderColumn = ( ) => (
-    <View>
-      {renderIdRow( )}
-      {renderCommentRow( )}
-      {renderQualityGrade( )}
-    </View>
-  );
-
-  const renderRow = ( ) => (
-    <View className="flex-row absolute bottom-1 justify-between w-full px-2">
-      <View className="flex-row">
-        {renderIdRow( )}
-        {renderCommentRow( )}
+  const iconColor = getIconColor( );
+  const margin = layout === "list" ? "mb-2.5" : "mr-2.5";
```

Comment:
Let's change the margin to `1`. I know that might not be what the issue said, but it looks closer to spec to me with `1`.